dist.ini

```diff
@@ -1,68 +1,12 @@
 name    = Finance-YieldCurve
-author  = binary.com <cpan@binary.com>
+author  = DERIV <DERIV@cpan.org>
 license = Perl_5
-copyright_holder = binary.com
-copyright_year   = 2017
-main_module	= lib/Finance/YieldCurve.pm
-
-[Git::GatherDir]
-exclude_filename = Makefile.PL
-include_dotfiles = 1
-[PruneCruft]
-except = t/rc/\.perl*rc$
-[ManifestSkip]
-[MetaYAML]
-[License]
-[Readme]
-[MakeMaker]
-eumm_version = 6.48
-prereq_fatal = 1
-[ExecDir]
-[ShareDir]
-dir = share
-[Manifest]
-[TestRelease]
-[ConfirmRelease]
-[UploadToCPAN]
-[Prereqs::FromCPANfile]
-[Prereqs / BuildRequires]
-perl = 5.010000
-[CheckPrereqsIndexed]
-[CheckExtraTests]
-[VersionFromModule]
-[PodVersion]
-[PkgVersion]
-[GitHub::Meta]
-repo = binary-com/perl-Finance-YieldCurve
-[InstallGuide]
-[MetaJSON]
-[InsertExample]
-[PodSyntaxTests]
-[MojibakeTests]
-[Test::CheckDeps]
-[Test::Compile]
-[Test::Synopsis]
-[Test::EOL]
-[Test::Version]
-[Test::Pod::LinkCheck]
-[PodCoverageTests]
-[Test::UnusedVars]
-[Test::ReportPrereqs]
-[SpellingCommonMistakesTests]
-[CopyFilesFromRelease]
-filename = lib/Finance/YieldCurve.pm
-filename = Makefile.PL
-;[Git::Check]
-;allow_dirty = dist.ini
-;changelog = Changes
-[Git::Commit]
-allow_dirty = dist.ini
-allow_dirty = cpanfile
-allow_dirty = Changes
-allow_dirty = Makefile.PL
-[Git::Tag]
-tag_format = v%v
-tag_message = Tag v%v for CPAN release
-[ReversionOnRelease]
-[NextRelease]
+copyright_holder = Deriv Services Ltd
```

Comment:
```suggestion
copyright_holder = Binary Services Ltd
```

Reply:
It has not been released to pause.cpan.org before. So it will be released to Deriv@pause.cpan.org